### PR TITLE
lydia-samir submit homework database week1

### PR DESCRIPTION
**Pull request by Lydia-GG**

### CODE REVIEW

File: Week1/ex1.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  console.log('meetup database created...');
+});
+
+connection.query('USE meetup', (error, results, fields) => {
+  if (error) throw error;
+  console.log('Use meetup database...');
+});
+
+connection.query(
+  'CREATE TABLE Invitee (invitee_no INT, invitee_name VARCHAR(100), invited_by VARCHAR(100))',
```
Comment (GinaSt) [inline Week1/ex1.js:30]: In this table you need a Primary key.

File: Week1/ex1.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+connection.query(
+  `INSERT INTO room (room_no, room_name, floor_number) VALUES (1,'Training Room', 1),(2,'Leadership',1),(3,'Conference',1),(4,'Triumph',2),(5,'Inspiration',2)`,
+  (error, results, fields) => {
+    if (error) throw error;
+    console.log('Room Rows has been added..');
+  }
+);
+
+connection.query(
+  'CREATE TABLE Meeting (meeting_no INT, meeting_title VARCHAR(100), starting_time TIME, ending_time TIME,room_no INT)',
```
Comment (GinaSt) [inline Week1/ex1.js:62]: Here there is a relationship between the Meeting table and the Room table. Is there a way to show this relationship. 
HINT: Try to add primary keys to both tables.

File: Week1/ex2.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  password: 'hyfpassword',
+  database: 'world',
+});
+
+connection.connect((err) => {
+  if (err) throw err;
+  console.log('MySQL connected...');
+});
+
+let queries = [
```
Comment (GinaSt) [inline Week1/ex2.js:15]: Nice and clean queries!

File: Week1/ex1.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+connection.query(
+  'CREATE TABLE Room (room_no INT, room_name VARCHAR(100),floor_number INT)',
+  (error, results, fields) => {
+    if (error) throw error;
+    console.log('Room table created...');
+  }
+);
+
+connection.query(
+  `INSERT INTO room (room_no, room_name, floor_number) VALUES (1,'Training Room', 1),(2,'Leadership',1),(3,'Conference',1),(4,'Triumph',2),(5,'Inspiration',2)`,
```
Comment (GinaSt) [inline Week1/ex1.js:54]: Nice that you insert multiple rows with one query!

File: Week1/ex1.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+connection.query(
+  `INSERT INTO invitee (invitee_no, invitee_name, invited_by) VALUES (1,'Maria','George'), (2,'Lydia','Heba'), (3,'Gloria','Danny'),(4,'John','Micheal'),(5,'Mark','Heidi')`,
+  (error, results, fields) => {
+    if (error) throw error;
+    console.log('Invitee Rows has been added..');
+  }
+);
+
+connection.query(
```
Comment (GinaSt) [inline Week1/ex1.js:45]: You could have a function to run all the create queries instead of rewriting the code.

**Review by GinaSt — CHANGES_REQUESTED** [review]
Hi Lydia! Your work is almost done! I see you have understood the basic SQL queries and how to create and populate a table. However, you could try to add primary and and foreign keys to your tables.

**Review by GinaSt — APPROVED** [review]
Nicely done :)

**Comment by Lydia-GG** [comment]
> Nicely done :)

thank you so much for your review and time @GinaSt. the code became more readable because of you .

**Pull request closed without merging**